schemas/definitions.json

```diff
@@ -229,8 +234,7 @@
 
     "url": {
       "type": "string",
-      "description": "A valid URL.",
-      "pattern": "^(http(s?):.*|mailto:.*|tel:(\\+?)[0-9]{4,})$"
+      "description": "A location of a resource, potentially relative. For hierarchical schemes (like http), applications must resolve relative URIs (e.g. './v1/api/'). Implementations should support the following schemes: http:, https:, mailto:, tel:, and ws:."
```

Comment:
@tkurki can you clarify the relative stuff? Is that for resources hosted by the server itself (and can non-hosted or resources on another server be non-relative?) Would we store the location of a USB device here as well? E.g. `file:///dev/tty.USB0`?